assets/style.css

```diff
@@ -0,0 +1,781 @@
+    width: 100%;
+  }
+
+  /* FOOTER */
+  .footer {
+    width: 100%;
+    height: 250px;
+    text-align: center;
+  }
+
+  .footer-links {
+    background-color: #131313;
+    padding: 2.5em 0;
+  }
+
+  .wrapper {
+    margin: 0 auto;
+    max-width: 100%;
+    padding: 0 20px;
+    height: 100%;
+  }
+
+  .footer-links-social,
+  .footer-links-social li a,
+  .footer-links-menu {
+    justify-content: center;
+  }
+
+  .footer-links-social {
+    font-size: 100%;
+    margin-bottom: 15px;
+    vertical-align: baseline;
+  }
+
+  .footer-links-social li a {
+    box-sizing: border-box;
+    background-color: #333;
+    margin: auto 15px;
+    border-radius: 20px;
+    padding: 10px;
+  }
+
+  .footer-links-menu {
+    margin-top: 50px;
+    font-size: 100%;
+    box-sizing: border-box;
+    align-items: center;
+  }
+
+  .footer-links-menu li a {
+    margin: 5px;
+    border: 0;
+    vertical-align: baseline;
+    box-sizing: border-box;
+    align-content: center;
+    flex-wrap: nowrap;
+  }
+
+  .footer-legal {
+    width: 100%;
+    height: 100px;
+    padding: 50px 0;
+    background-color: #000;
+    display: block;
+  }
+
+  span img {
+    width: 37px;
+    height: 9px;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  /* ====== NAVIGATION ====== */
+  .logo-div,
+  .fa-bars {
+    display: none;
+  }
+
+  .width-1024 {
+    display: block;
+  }
+
+  .width-1023 {
+    display: block;
+  }
+
+  .width-320 {
+    display: none;
+  }
+
+  .first-navbar {
+    background-color: #131313;
+  }
+
+  .split-links,
+  .second-nav-links,
+  .latest-funding-title-links,
+  .footer-links {
+    justify-content: space-between;
+  }
+
+  .nav-flex {
+    width: 100%;
+  }
+
+  .funding-links-right li {
+    padding: 0 5px;
+    border-right: 1px solid #fe4736;
+  }
+
+  .first-navbar ul li {
+    margin-right: 10px;
+  }
+
+  .first-navbar li a {
+    color: #555;
+  }
+
+  .news {
+    color: #fff !important;
+  }
+
+  .first-nav-ul {
+    padding-left: 10px;
+  }
+
+  /* second-nav-bar */
+
+  .second-nav-links-span span {
+    margin-right: 10px;
+  }
+
+  .second-nav-links-span span a {
+    color: rgb(146, 140, 140);
+  }
+
+  .second-nav-links {
+    width: 97%;
+    font-size: 14px;
+    padding: 18px 12px 0;
+    position: relative;
+  }
+
+  .lens-menu {
+    background-color: red;
+    position: absolute;
+    right: 0;
+  }
+
+  .fa-search {
+    position: absolute;
+    right: 10px;
+  }
+
+  /* SHOWCASE */
+  .showcase-section {
+    grid-template-columns: 70% 30%;
+    gap: 1em;
+  }
+
+  .showcase-logo-div {
+    top: -140px;
+  }
+
+  .showcase-logo {
+    display: inline-block;
+    width: 400px;
+  }
+
+  .release-date {
+    float: right;
+    bottom: 150px;
+    right: 20px;
+  }
+
+  .mobile-showcase-content .release-date {
+    display: block;
+  }
+
+  .main-showcase {
+    height: 400px;
+  }
+
+  .aside-showcase {
+    height: 47%;
+  }
+
+  .main-showcase > div h1 {
+    font-size: 40px;
+  }
+
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<h1>` element` and delete the left unnecessary rules

**Example:**
```CSS
/* before */
.card-1-mobile-text h3 a {

/* after */
.anchortag-class {
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_